beets/util/artresizer.py

```diff
@@ -190,14 +191,14 @@ def __init__(self):
                 self.im_convert_cmd = ['magick']
                 self.im_identify_cmd = ['magick', 'identify']
 
-    def resize(self, maxwidth, path_in, path_out=None):
+    def resize(self, maxwidth, quality, path_in, path_out=None):
```

Comment:
I think it is not a good idea to change the function signature here. Rather, `quality` should be appended as a keyword argument with a reasonable default value, as you did in `art.py`. I'm quite certain that otherwise, we will break third-party plugins without a good reason.